src/components/CDPList.js

```diff
@@ -42,13 +42,13 @@ const NavbarItem = ({ href, label, ratio, owned, active, ...props }) => (
 
 const CDPList = memo(function({ currentPath, viewedAddress, currentQuery }) {
   const { maker, account } = useMaker();
-  const [{ cdps, feeds }] = useStore();
+  const [{ urns, ilks }, dispatch] = useStore();
   const [ratios, setRatios] = useState([]);
   const [navbarCdps, setNavbarCdps] = useState([]);
 
   useEffect(() => {
     if (account) {
-      account.cdps.forEach(cdp => trackCdpById(maker, cdp.id));
+      account.cdps.forEach(cdp => trackCdpById(maker, cdp.id, dispatch));
```

Comment:
trackCdpById gets a dispatch argument so that we can store the cdp => ilk mapping as soon as we have it